google/cloud/forseti/common/util/rule_validator.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
```

Comment:
Will this module have a use case outside of the firewall enforcer? If not, it might be more clear to keep this file within the enforcer directory. "Rules" can mean multiple things, so it isn't 100% clear that this is for firewall rules (possibly only within enforcer).

Reply:
Yes, it is useful for systems that write the rules for forseti to apply. This can be a useful CI/CD test and/or as a sanity check for projects like capirca (https://github.com/google/capirca/tree/master/policies/pol)

Reply:
That said, I think adding more rule validation code here would be appropriate.